src/components/menu/Menu.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Pressable, StyleSheet, View, ViewStyle } from 'react-native';
+import { Menu as BaseMenu } from 'react-native-paper';
+import { useTheme } from '../../styles/themes';
+import { Icon } from '../icons/Icon';
+import type { IconName } from '../icons/IconProps';
+
+type MenuOption = {
+    title: string;
+    onPress: () => void;
+};
+
+interface Props {
+    visible: boolean;
+    openMenu: () => void;
+    closeMenu: () => void;
```

Comment:
```suggestion
    onPress: () => void;
    onDismiss: () => void;
```

I think the prop names should not reflect what they should do (in this case open the menu or close the menu) because it is a prop it means that when the caller will use it could do something else than open or close the menu. Instead they should represent what interaction has been done.

Reply:
Yes, you are right ! fixed